### DISCUSSION TITLE
Spa Pump S200103 SANTORINI

### DISCUSSION THREAD

**Discussion by danielsoar** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#443] [General]
Hi Guys,

I have the boards and I am just about to connect the JST leads, have done a little reading and there are quite a few views.

I have two fully loaded BWC v2 boards with LPF installed, I guess the first try is to connect on H1A and H2A without any resistors but after reading all the way through issue #330 I can see its closed but cant work out what the final solution was.

I will assume that my pump is similar to S200102 to start with and load maldives2021 since it has hydrojets, only difference I can see in the spa spec is that this unit has 10 jets instead of 8 so perhaps it has a slightly larger pump (or perhaps not, maybe its just a revision of the last version)

Any photos I should take while inside, and what is the spaghetti wiring people mention, is that where we are crossing over D3 and D6 wires on the connectors?

I am just having trouble figuring out in my head what step 1, 2, 3 should be in terms of wiring configs to try.

I have some old NodeMCUs laying around as well as a newer V3 as well as both the red and blue LLC's (ill start with the red)

From what I can see nobody has reported trying the Santorini spa yet - or at least I cant find reference to the S200103 pump model number anywhere

**Comment by danielsoar** [comment]
![20230205_111324](https://user-images.githubusercontent.com/6870343/217415829-ecdef34e-c8d6-4043-8eeb-5a9a603ec6ec.jpg)
![20230208_122413](https://user-images.githubusercontent.com/6870343/217415860-bf56f66b-178d-40fd-ae88-d01e647bf75e.jpg)

**Comment by visualapproach** [comment]
Here is a nice write up #437
